pkg/bootstrap/bootstrap.go

```diff
@@ -0,0 +1,69 @@
+package bootstrap
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/weaveworks/pctl/pkg/runner"
+	profilesv1 "github.com/weaveworks/profiles/api/v1alpha1"
+	"gopkg.in/yaml.v2"
+)
+
+type Config struct {
+	// GitRepository is the git repository flux resource the installation uses
+	GitRepository profilesv1.GitRepository `json:"gitRepository,omitempty"`
+}
+
+var r runner.Runner = &runner.CLIRunner{}
+
+func CreateConfig(namespace, name, directory string) error {
+	out, err := r.Run("git", "-C", directory, "rev-parse", "--show-toplevel")
+	if err != nil {
+		if strings.Contains(string(out), "not a git repository") {
```

Comment:
We could check this ourselves earlier when the command is called by simply doing an os.Stat on `.git` folder.

Reply:
this approach it also finds the path to the top-level directory for you. You can run bootstrap from within the a sub directory and it will still work. I think its nice/cleaner than looking for `.git` dir